dotCMS/src/integration-test/java/com/dotcms/contenttype/test/ContentTypeAPIImplTest.java

```diff
@@ -955,47 +793,34 @@ public void testFormContentTypeWithPublishExpireFields() throws Exception{
 		long time = System.currentTimeMillis();
-		
-		ContentType contentType = ContentTypeBuilder
-                .builder(BaseContentType.getContentTypeClass(base))
-                .description("PersonaContentTypeWithPublishExpireFields " + time)
-                .folder(FolderAPI.SYSTEM_FOLDER)
-                .host(Host.SYSTEM_HOST)
-                .name("PersonaContentTypeWithPublishExpireFields " + time)
-                .owner(APILocator.systemUser().toString())
-                .variable("PersonaCTVariable")
-                .publishDateVar("publishDate")
-                .expireDateVar("expireDate")
-                .build();
-        contentType = contentTypeApi.save(contentType);
 
-        assertThat("ContentType exists", contentTypeApi.find( contentType.inode() ) != null);
-        
-        List<Field> fields = new ArrayList<>( contentType.fields() );
-        
-        Field fieldToSave = FieldBuilder.builder( DateTimeField.class )
-                .name( "Publish Date" )
-                .variable( "publishDate" )
-                .contentTypeId( contentType.id() )
-                .dataType( DataTypes.DATE )
-                .indexed(true)
-                .build();
-        fields.add( fieldToSave );
-        
-        fieldToSave = FieldBuilder.builder( DateTimeField.class )
-                .name( "Expire Date" )
-                .variable( "expireDate" )
-                .contentTypeId( contentType.id() )
-                .dataType( DataTypes.DATE )
-                .indexed(true)
-                .build();		
-        fields.add( fieldToSave );
-        
-        contentType = contentTypeApi.save(contentType, fields);
-        
-      //Deleting content type.
-      	delete(contentType);
-        
+		ContentType contentType = ContentTypeBuilder.builder(BaseContentType.getContentTypeClass(base))
+				.description("ContentTypeWithPublishExpireFields " + time).folder(FolderAPI.SYSTEM_FOLDER)
+				.host(Host.SYSTEM_HOST).name("ContentTypeWithPublishExpireFields " + time)
+				.owner(APILocator.systemUser().toString()).variable("CTVariable").publishDateVar("publishDate")
+				.expireDateVar("expireDate").build();
+		contentType = contentTypeApi.save(contentType);
+
+		assertThat("ContentType exists", contentTypeApi.find(contentType.inode()) != null);
+
+		List<Field> fields = new ArrayList<>(contentType.fields());
+
+		Field fieldToSave = FieldBuilder.builder(DateTimeField.class).name("Publish Date").variable("publishDate")
+				.contentTypeId(contentType.id()).dataType(DataTypes.DATE).indexed(true).build();
+		fields.add(fieldToSave);
+
+		fieldToSave = FieldBuilder.builder(DateTimeField.class).name("Expire Date").variable("expireDate")
+				.contentTypeId(contentType.id()).dataType(DataTypes.DATE).indexed(true).build();
+		fields.add(fieldToSave);
+
+		contentType = contentTypeApi.save(contentType, fields);
+
+		// Deleting content type.
+		delete(contentType);
```

Comment:
Looks nice! very small test methods now. I would also and a try - finally (no catch) and put the deletion of the content type inside the finally, because if an assert fails or an exception is thrown, the deletion won't occur and the content type will remain undeleted